src/js/utils/StringUtil.js

```diff
@@ -1,6 +1,14 @@
 import marked from 'marked';
 
 const StringUtil = {
+  arrayToCommaSeparatedString: function (array) {
```

Comment:
I don't see a huge need for this function, to be honest – but I guess we could make it a tad more generic for example by having the `separator` be an optional argument?
How about `arrayToJoinedString(array = [], separator = ',') { ` ?